about/complying_with_licenses.rst

```diff
@@ -36,7 +41,7 @@ Requirements
 In the case of the MIT license, the only requirement is to include the license
 text somewhere in your game or derivative project.
 
-This text reads as follows:
+This text reads as follows::
```

Comment:
Makes it a code block, so that it's not extracted for translation.

Reply:
Hmm, but with the added line breaks, that makes it slightly more annoying to reuse in a project...

Reply:
I'd say it's a necessary evil in this case :)